server/server.go

```diff
@@ -330,6 +330,10 @@ type stats struct {
 	inBytes       int64
 	outBytes      int64
 	slowConsumers int64
```

Comment:
Maybe make its own struct, slowConsumers and inside is total, client, routes, etc..

Reply:
put the slow consumer counters in their own struct using atomic.Uint64 instead, thanks for the suggestion 👍

Reply:
ah but atomic.Uint64 is only available in +Go 1.19... and we still have Go 1.18 in Travis right now...

Reply:
rebased and updated Go 1.19 as the minimum supported version for dev branch.